torch/csrc/jit/tensorexpr/operators/misc.cpp

```diff
@@ -160,7 +159,7 @@ ExprHandle demoteOutput(
   return e;
 }
 
-std::optional<TensorInfo> getTensorInfo(BufHandle b) {
+std::optional<TensorInfo> getTensorInfo(const BufHandle& b) {
   std::vector<int64_t> dims;
```

Comment:
Add a reserve here while you are editing this file

Reply:
Added